### ISSUE TITLE
Adjusting for ambient noise in input audio recording

### ISSUE THREAD

**Issue by bmanczak** [facebookresearch/flashlight#476]
### Question

Do flashlight audio models adjust the energy threshold of the audio dynamically to account for background noise? i.e. can one input the audio with noise and expect reasonable performance?

#### Additional Context
I am using the flashlight speech2text model from this [paper](https://arxiv.org/pdf/2012.03411.pdf) on raw audio extracted from MS Teams meeting. Since the properties of the raw audio are quite different than the training data (audiobooks) I am wondering whether normalisation should be done prior to feeding into to the model. In particular, I would like to adjust for ambient noise in the recordings. Neither in code nor in the paper could I find whether during featurization this is done. 

I've found [this](https://github.com/facebookresearch/flashlight/tree/master/bindings/python#featurization) python binding for featurization and I see `params.use_energy` setting that might be related to the adjustment I am looking for. However, I do not know if for my use-case I should use it and if yes then how.

**Comment by tlikhomanenko** [comment]
cc @xuqiantong @vineelpratap @syhw on details.

**Comment by vineelpratap** [comment]
Hi, 
  We don't have an option to adjust for ambient noise in the recordings. You would have to either re-train the model with noise augmentation or run a denoiser on the audio file before running inference.
